src/DataStructures/InfluxDatapoint.cs

```diff
@@ -142,7 +148,11 @@ public string ConvertToInfluxLineProtocol()
             else if (tType == typeof(double))
                 ////double has to have a . as decimal seperator for Influx
                 fields = String.Join(",", Fields.Select(v => new StringBuilder().AppendFormat("{0}={1}", v.Key.EscapeChars(comma: true, equalSign: true, space: true), String.Format(new CultureInfo("en-US"), "{0}", v.Value))));
-            else if (tType == typeof(InfluxValueField))
+#if NETSTANDARD1_6
```

Comment:
If we can update to .net standard 2.0 is is not necessary anymore!

Reply:
i think we need to update it to .net standard 2, its been a while. I still support .Net 4.5 on the non core side :)

Reply:
With .Net Standard 2.0 the project seperation is not required enymore!?! The library can be used easy with .net 4.6.1 onwoard.

Reply:
I would suggest to create a last nuget for 
AdysTech.InfluxDB.Client.Net and add a dependency to the AdysTech.InfluxDB.Client.Net.Core
Projekt. In this way only one nuget is relevant. What you think?